resources/fixtures/config/barcode.yml

```diff
@@ -1,3 +1,9 @@
 file-type: 'png'
 barcode-type: 'code39'
+
+# determines the class that will be used to automatically assign barcodes to products, will typically be the same as
+# the barcode-type
+barcode-generator: 'code39'
```

Comment:
Is there a good reason why this and the barcode-type configs are separate. This could get confusing, and i don't see a reason why they *would* be different.